web/application.html

```diff
@@ -0,0 +1,209 @@
+        }
+        canvas.emscripten {
+            border: none;
+        }
+        .centered {
+            position: absolute;
+            top: 50%;
+            left: 50%;
+            -moz-transform: translateX(-50%) translateY(-50%);
+            -webkit-transform: translateX(-50%) translateY(-50%);
+            transform: translateX(-50%) translateY(-50%);
+        }
+    </style>
+</head>
+<body>
+    <div class="centered">
+        <div class="emscripten" id="status">Downloading...</div>
+        <progress value="10" max="100" id="progress" hidden></progress>
+    </div>
+    <div id="container">
+        <canvas class="emscripten" id="canvas" oncontextmenu="event.preventDefault()" tabindex=-1></canvas>
+        <div class="emscripten fulscreen-container" id="fullscreen" style="display: none;">
+            <input type="button" value="Go Fullscreen" onclick="viewFullScreen()">
+        </div>
+    </div>
+    <script type='text/javascript'>
+        var canvasElement = document.getElementById('canvas');
+        var devicePixelRatio = window.devicePixelRatio || 1;
+
+        // Detect the visible width and height of the window
+        var canvasWidth = (screen.availWidth - (window.outerWidth - window.innerWidth)) * devicePixelRatio;
+        var canvasHeight = (screen.availHeight - (window.outerHeight - window.innerHeight)) * devicePixelRatio;
+
+        // Detect fullscreen change and resize canvas resolution accordingly
+        function fullscreenHandler(event) {
+            if (Module['JSCanvasSize']) {
+                if (isFullScreen()) {
+                    Module.JSCanvasSize(screen.width * devicePixelRatio, screen.height * devicePixelRatio, true, devicePixelRatio);
+                } else {
+                    if (document.referrer.includes('itch.io')) {
+                        canvasWidth = parseInt(window.getComputedStyle(canvasElement).getPropertyValue('width')) * devicePixelRatio;
+                        canvasHeight = parseInt(window.getComputedStyle(canvasElement).getPropertyValue('height')) * devicePixelRatio;
+                    }
+                    Module.JSCanvasSize(canvasWidth, canvasHeight, false, devicePixelRatio);
+                }
+            }
+        }
+
+        document.addEventListener('fullscreenchange', fullscreenHandler, false);
+        document.addEventListener('mozfullscreenchange', fullscreenHandler, false);
+        document.addEventListener('webkitfullscreenchange', fullscreenHandler, false);
+        document.addEventListener('MSFullscreenChange', fullscreenHandler, false);
+
+        // When window size has changed, resize the canvas accordingly
+        window.addEventListener('resize', function(evt) {
+            canvasWidth = (screen.availWidth - (window.outerWidth - window.innerWidth)) * devicePixelRatio;
+            canvasHeight = (screen.availHeight - (window.outerHeight - window.innerHeight)) * devicePixelRatio;
+            fullscreenHandler(evt);
```

Comment:
`fullscreenHandler` name isn't really appropriate if it's called on resize too.
Maybe rename it? `viewportResizeHandler`, or whatever you like.

Reply:
`viewportResizeHandler ` it is